aws-aurora-mysql/README.md

```diff
@@ -57,6 +57,7 @@ No provider.
 | instance\_count | Number of instances to create in this cluster. | `string` | `1` | no |
 | kms\_key\_id | If provided, storage will be encrypted with this key, otherwise an AWS-managed key is used. (Encryption is always on). | `string` | `""` | no |
 | owner | Owner for tagging and naming. See [doc](../README.md#consistent-tagging). | `string` | n/a | yes |
+| params\_engine\_version | n/a | `string` | `"5.7"` | no |
```

Comment:
mind including a description for this parameter?

Reply:
Note that you have to write the description in the terraform code and run `make docs`.